### ISSUE TITLE
Use thumbnails for /snaps

### ISSUE THREAD

**Issue by bonny** [EarthPeople/PushSnapper#1]
The /snaps URL displays all snaps using the full width images. Because the full size images are used the page can take a long time to load. We should display thumbnails for all images instead. https://github.com/lovell/sharp could be used for this.

**Comment by carlcalderon** [comment]
I agree. The `/snaps` page should display thumbnails and link to the full image.
